stable/redis-ha/templates/redis-sentinel-deployment.yaml

```diff
@@ -3,23 +3,14 @@ kind: Deployment
 metadata:
   name: {{ template "fullname" . }}-sentinel
 spec:
-  replicas: {{ .Values.replicas.sentinel }}
+  replicas: {{ .Values.replicas.sentinels }}
   template:
     metadata:
       labels:
-        name: redis-sentinel
-        redis-sentinel: "true"
-        role: sentinel
+        app: "redis-ha"
+        name: "redis-sentinel"
```

Comment:
Same as previous comment, it's customary to have the fullname prefixing names of resources, as that allows one to have multiple setups.

Reply:
Same as above